test/e2etest/installer_test.go

```diff
@@ -9,12 +10,14 @@ import (
 func TestInstaller(t *testing.T) {
 
 	// List of the products to test
-	productList := []string{jira, confluence, bamboo}
+	productList := []string{jira, confluence, bamboo, bitbucket}
 	testConfig := createConfig(t, productList)
 
 	// Schedule uninstall and cleanup the environment
 	defer runUninstallScript(testConfig.ConfigPath)
 
+	println(fmt.Sprintf("################## AWS REGION '%s' ##################", testConfig.AwsRegion))
```

Comment:
When we started doing our own test output formatting, can we put a similar line between each product assertion?

Reply:
Added this line for each product test suite